Tasks/SshV0/task.json

```diff
@@ -16,7 +16,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
```

Comment:
Is it possible for this change to break existing build definitions?  Could a user be relying on wrong behavior?  If so, we may want to bump the major version number.

Reply:
I thought about it, I think it is better to not break major version here. There is a way to unblock users relying on this behavior but that would be strange.